=== RUN  TestEqual
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$equal_equal (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 12151 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testequal_testEqual [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:11507:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestEqual

=== RUN  TestEqualFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$equalfunc_equalFunc (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 12300 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testequalfunc_testEqualFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:12226:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestEqualFunc

=== RUN  TestCompare
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$compare_compare (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 13454 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testcompare_testCompare [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:13401:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestCompare

=== RUN  TestCompareFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$_$equaltocmp__equalToCmp (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 13578 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testcomparefunc_testCompareFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:13468:90)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestCompareFunc

=== RUN  TestIndex
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$index_index (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14494 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testindex_testIndex [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14485:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestIndex

=== RUN  TestIndexFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$_$equaltoindex__equalToIndex (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14531 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testindexfunc_testIndexFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14502:75)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestIndexFunc

=== RUN  TestContains
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$contains_contains (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14548 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testcontains_testContains [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14539:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestContains

=== RUN  TestContainsFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$_$equaltoindex__equalToIndex (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14531 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testcontainsfunc_testContainsFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14556:81)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestContainsFunc

=== RUN  TestInsert
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$insert_insert (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14637 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testinsert_testInsert [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14594:13)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestInsert

=== RUN  TestInsertOverlap
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$insert_insert (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14637 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testinsertoverlap_testInsertOverlap [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14800:63)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestInsertOverlap

=== RUN  TestDelete
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testdelete_testDelete [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14821:15)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestDelete

=== RUN  TestDeleteFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testdeletefunc_testDeleteFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:14841:15)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestDeleteFunc

=== RUN  TestDeletePanics

-- PASS: TestDeletePanics

=== RUN  TestClone
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testclone_testClone [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15116:12)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestClone

=== RUN  TestCompact
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testcompact_testCompact [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15167:15)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestCompact

=== RUN  TestCompactFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testcompactfunc_testCompactFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15187:15)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestCompactFunc

=== RUN  TestGrow
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testgrow_testGrow [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15216:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestGrow

=== RUN  TestClip
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testclip_testClip [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15345:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestClip

=== RUN  TestReverse
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$reverse_reverse (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 15413 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testreverse_testReverse [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15371:2)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestReverse

=== RUN  TestReplace
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testreplace_testReplace [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:15964:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestReplace

=== RUN  TestReplacePanics
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testreplacepanics_testReplacePanics [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16005:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestReplacePanics

=== RUN  TestReplaceOverlap
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$replace_replace (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 15989 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testreplaceoverlap_testReplaceOverlap [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16043:66)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestReplaceOverlap

=== RUN  TestRotate
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$_$rotateleft__rotateLeft (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16087 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testrotate_testRotate [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16073:4)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestRotate

=== RUN  TestInsertGrowthRate
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$insert_insert (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14637 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testinsertgrowthrate_testInsertGrowthRate [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16095:8)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestInsertGrowthRate

=== RUN  TestReplaceGrowthRate
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$replace_replace (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 15989 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testreplacegrowthrate_testReplaceGrowthRate [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16729:8)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestReplaceGrowthRate

=== RUN  TestInference
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$_$apply__apply (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16762 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testinference_testInference [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16743:2)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestInference

=== RUN  TestIntSorts
Exception: Cannot read properties of null (reading 'value')
Called from module TypeError: Cannot read properties of null (reading 'value')
Called from stdgo__$internal_unsafe_UnsafePointer.__toRef__ (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14026 column 15)
Called from stdgo__$internal_sync_atomic_$_Pointer_$_$static_$extension.load (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16915 column 47)
Called from .stdgo__$internal_math_rand_Rand_$seed_seed (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16790 column 74)
Called from .stdgo__$internal_slices_Slices_$_$makerandomints__makeRandomInts (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16780 column 2)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testintsorts_testIntSorts [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:16765:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)

-- FAIL: TestIntSorts

=== RUN  TestStringSorts
Exception: Cannot read properties of null (reading 'value')
Called from module TypeError: Cannot read properties of null (reading 'value')
Called from stdgo__$internal_unsafe_UnsafePointer.__toRef__ (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14026 column 15)
Called from stdgo__$internal_sync_atomic_$_Pointer_$_$static_$extension.load (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16915 column 47)
Called from .stdgo__$internal_math_rand_Rand_$seed_seed (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16790 column 74)
Called from .stdgo__$internal_slices_Slices_$_$makerandomstrings__makeRandomStrings (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 22315 column 2)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$teststringsorts_testStringSorts [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22300:12)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)

-- FAIL: TestStringSorts

=== RUN  TestStructSorts
Exception: Cannot read properties of null (reading 'value')
Called from module TypeError: Cannot read properties of null (reading 'value')
Called from stdgo__$internal_unsafe_UnsafePointer.__toRef__ (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14026 column 15)
Called from stdgo__$internal_sync_atomic_$_Pointer_$_$static_$extension.load (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16915 column 47)
Called from .stdgo__$internal_math_rand_Rand_$seed_seed (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 16790 column 74)
Called from .stdgo__$internal_slices_Slices_$_$makerandomstructs__makeRandomStructs (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 22527 column 2)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$teststructsorts_testStructSorts [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22434:12)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)

-- FAIL: TestStructSorts

=== RUN  TestSortIntSlice
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testsortintslice_testSortIntSlice [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22640:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestSortIntSlice

=== RUN  TestSortFuncIntSlice
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testsortfuncintslice_testSortFuncIntSlice [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22691:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestSortFuncIntSlice

=== RUN  TestSortFloat64Slice
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testsortfloat64slice_testSortFloat64Slice [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22741:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestSortFloat64Slice

=== RUN  TestSortFloat64SliceWithNaNs
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testsortfloat64slicewithnans_testSortFloat64SliceWithNaNs [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22796:15)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestSortFloat64SliceWithNaNs

=== RUN  TestSortStringSlice
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$clone_clone (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 14640 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testsortstringslice_testSortStringSlice [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22883:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestSortStringSlice

=== RUN  TestSortLarge_Random
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$issorted_isSorted (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 22654 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testsortlarge_$random_testSortLarge_Random [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22911:5)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestSortLarge_Random

=== RUN  TestStability
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$issortedfunc_isSortedFunc (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 23002 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$teststability_testStability [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:22946:5)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestStability

=== RUN  TestMinMax
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$minfunc_minFunc (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 35833 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testminmax_testMinMax [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:23098:12)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestMinMax

=== RUN  TestMinMaxNaNs
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$min_min (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 35830 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testminmaxnans_testMinMaxNaNs [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:35879:27)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestMinMaxNaNs

=== RUN  TestMinMaxPanics

-- PASS: TestMinMaxPanics

=== RUN  TestBinarySearch

-- PASS: TestBinarySearch

=== RUN  TestBinarySearchInts

-- PASS: TestBinarySearchInts

=== RUN  TestBinarySearchFloats

-- PASS: TestBinarySearchFloats

=== RUN  TestBinarySearchFunc
Exception: generic function is not supported
Called from .stdgo__$internal_slices_Slices_$binarysearchfunc_binarySearchFunc (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 36011 column 23)
Called from module at stdgo__$internal_testing_InternalTest.stdgo__$internal_slices_Slices_$testbinarysearchfunc_testBinarySearchFunc [as f] (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:36129:16)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 45918 column 9)
Called from ._$internal_slices_$dot_$test_Slices_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_slices.js line 38656 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_slices.js:55386:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_slices.js:55387:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1734 column 14)
Called from Object..js (node:internal/modules/cjs/loader line 1899 column 10)

-- FAIL: TestBinarySearchFunc

